client/my-sites/site-settings/controller.js

```diff
@@ -85,32 +79,11 @@ const controller = {
 		}
 	},
 
-	siteSettings( context ) {
-		let analyticsPageTitle = 'Site Settings';
-		const basePath = route.sectionify( context.path );
-		const siteId = getSelectedSiteId( context.store.getState() );
-		const section = sectionify( context.path ).split( '/' )[ 2 ];
-		const state = context.store.getState();
-
-		// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
-		context.store.dispatch( setTitle( i18n.translate( 'Site Settings', { textOnly: true } ) ) );
-
-		// if site loaded, but user cannot manage site, redirect
-		if ( siteId && ! canCurrentUser( state, siteId, 'manage_options' ) ) {
-			page.redirect( '/stats' );
-			return;
-		}
-
+	general( context ) {
 		renderPage(
 			context,
-			<SiteSettingsComponent section={ section } />
+			<SiteSettingsMain />
 		);
-
-		// analytics tracking
-		if ( 'undefined' !== typeof section ) {
-			analyticsPageTitle += ' > ' + titlecase( section );
-		}
-		analytics.pageView.record( basePath + '/:site', analyticsPageTitle );
```

Comment:
It's recorded in the shared settings controller, see https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/site-settings/settings-controller.js#L36

Side note: we might want to alter these to use data components, similar to how we did with `setDocumentHeadTitle` and `<DocumentHead />`.